extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -177,27 +184,160 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 
 	public async getDatabaseAssessments(): Promise<ServerAssessement> {
 		const ownerUri = await azdata.connection.getUriForConnection(this.sourceConnectionId);
-		// stress test backend & dialog component
-		const assessmentResults = await this.migrationService.getAssessments(
-			ownerUri,
-			this._databaseAssessment
-		);
-		const dbAssessments = assessmentResults?.assessmentResult.databases.filter(d => !this.excludeDbs.includes(d.name)).map(d => {
-			return {
-				name: d.name,
-				issues: d.items.filter(i => i.appliesToMigrationTargetPlatform === MigrationTargetType.SQLMI) ?? []
-			};
-		});
+		this._assessmentApiResponse = (await this.migrationService.getAssessments(ownerUri, this._databaseAssessment))!;
 		this._assessmentResults = {
-			issues: assessmentResults?.assessmentResult.items?.filter(i => i.appliesToMigrationTargetPlatform === MigrationTargetType.SQLMI) ?? [],
-			databaseAssessments: dbAssessments! ?? []
+			issues: this._assessmentApiResponse.assessmentResult.items,
+			databaseAssessments: this._assessmentApiResponse.assessmentResult.databases.map(d => {
+				return {
+					name: d.name,
+					issues: d.items
+				};
+			})
 		};
-
+		this.generateAssessmentTelemetry();
```

Comment:
Comment explaining that you're purposely not wanting to block on sending telemetry